### PR TITLE
Adding Phidgets Sample

### PR DESCRIPTION
**Pull request by anthony-ngu**

### CODE REVIEW

File: PhidgetsSensors.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+1. Hook up the Sound Sensor to a port on the Phidget I/O Board.
+
+# Create a new project
+
+1. Create a new project from the template.
+1. Go to the Quick Downloads section at the [Phidgets site](http://www.phidgets.com/docs/OS_-_Windows#Quick_Downloads){:target="_blank"}.
+1. Click on the "Phidget21 Libraries" link to download the libaries, unzip the folder, and use the dll and lib under phidget21-windevel\IA32 when compiling your project.
+1. Copy the phidget21.h file into your project.
+1. Link your project to the phidget21.lib.
+1. Place the phidget21.dll into your Galileo's Windows\System32 folder so that any future program using Phidget code can run.
```
Comment (ooeygui) [inline PhidgetsSensors.md:28]: Should it be Phidget or Phidgets?

File: PhidgetsSensors.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+	//Registers a callback that will run if the sensor value changes by more than the OnSensorChange trig-ger.
+	//Requires the handle for the IntefaceKit, the function that will be called, and an arbitrary pointer that will be supplied to the callback function (may be NULL).
+	CPhidgetInterfaceKit_set_OnSensorChange_Handler(ifKit, SensorChangeHandler, NULL);
+
+	//open the interfacekit for device connections
+	CPhidget_open((CPhidgetHandle)ifKit, -1);
+
+	//get the program to wait for an interface kit device to be attached
+    Log(L"Waiting for interface kit to be attached....\n");
```
Comment (ooeygui) [inline PhidgetsSensors.md:109]: Odd tabbing?

**Comment by ooeygui** [comment]
Minor comments. Otherwise, looks good.

File: PhidgetsSensors.md (hunk 1 above)
Comment (anthony-ngu) [inline PhidgetsSensors.md:28]: On their site they use Phidgets for plural and Phidget for singular

**Pull request merged**